types/esri-leaflet/index.d.ts

```diff
@@ -596,6 +611,153 @@ declare module 'leaflet' {
         type FeatureCallbackHandler = (error?: any, featureCollection?: any, response?: any) => void;
         type ResponseCallbackHandler = (error?: any, response?: any) => void;
 
+        /**
+         * Options for `L.esri.ImageMapLayer`
+         */
+        interface ImageMapLayerOptions extends ImageOverlayOptions, LayerOptionsBase {
+            /**
+             * If there are multiple bands, you can specify which bands to export
+             */
+            bandIds?: string | undefined;
+
+            /**
+             * The pixel value representing no information.
+             */
+            noData?: number | undefined;
+
+            /**
+             * Interpretation of the noData setting.
+             */
+            noDataInterpretation?: string | undefined;
+
+            /**
+             * Leave pixelType as unspecified, or UNKNOWN, in most exportImage use cases, unless such pixelType is
+             * desired. Possible values: C128, C64, F32, F64, S16, S32, S8, U1, U16, U2, U32, U4, U8, UNKNOWN.
+             */
+            pixelType?: string | undefined;
+
+            /**
+             * A JSON representation of a raster function
+             */
+            renderingRule?: object | undefined;
+
+            /**
+             * A JSON representation of a mosaic rule
+             */
+            mosaicRule?: object | undefined;
+        }
+
+        /**
+         * Render and visualize Image Services from ArcGIS Online and ArcGIS Server.
+         * Image Services provide access to raster data through a web service.
+         */
+
+        class ImageMapLayer extends RasterLayer {
+            constructor(options: ImageMapLayerOptions);
+
+            /**
+             * 	Uses the provided function to create a popup that will identify pixel value(s) whenever the map is clicked.
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/esri-leaflet/index.d.ts;
  Line: 659;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.